thrimshim/thrimshim/main.py

```diff
@@ -138,15 +138,15 @@ def update_row(ident, new_row):
 	new_row['error'] = None
 
 	# actually update database
-	build_query = sql.SQL("""
+	query_str = """
```

Comment:
Use `sql.SQL().format()`, not `string.format()`, for formatting SQL queries. String formatting makes it way too easy to accidentially put in an SQL injection.